rotkehlchen/api/v1/resources.py

```diff
@@ -1455,23 +1511,22 @@ def make_put_schema(self) -> EvmAccountsPutSchema:
         )
 
     @require_loggedin_user()
-    @resource_parser.use_kwargs(make_put_schema, location='json_and_query')
-    def put(
-            self,
-            accounts: list[dict[str, Any]],
-            async_query: bool,
-    ) -> Response:
+    @resource_parser.use_args(
+        make_put_schema,
+        location='json_and_query',
+        allow_async_validation=True,
+        as_kwargs=True,
+    )
+    @allow_async_validation()
```

Comment:
Can't this somehow be combined with the above? You need both to make it work. Right now it's readable, all good so no need to press it. Just wonder if it can be combined.

Reply:
I was thinking on doing it but is hard for two reasons:

1. You need the ApiServer that lives in the view method since is the one that has access to the "task manager"
2. The resource parser is already doing some "magic" and adding more things there complicated the logic